### PR TITLE
Added flowrate_counters

### PR DESCRIPTION
**Pull request by pranay2811**
New Telemetry fields: gas_flowrate_counter, water_flowrate_counter 
New Canonical Type: WM_PULSE

### CODE REVIEW

**Comment by pranay2811** [comment]
@shambergoldstein I added a new canonical type WM_PULSE, but I noticed that the guid is not generating. All the required settings are ON for auto-generating GUIDs. Is there something that I missed?

**Comment by shambergoldstein** [comment]
Make sure in your fork settings under Actions > General you have "Allow all actions and reusable workflows" checked under Actions permissions and "Read and write permissions" as well as "Allow Github Actions to create and approve pull requests" checked under Workflow permissions. Click save then go to Actions (outside of settings) and make sure you've clicked "I understand my workflows, go ahead and enable them". If it still isn't working I would reach out to Github for help and input guids manually using a generator like https://www.uuidgenerator.net/guid

File: ontology/yaml/resources/METERS/entity_types/WM.yaml [hunk 1]
```diff
@@ -59,4 +59,15 @@ WM_WASTE_NONCANONICAL_1:
   - water_volume_accumulator
   - mixing_water_tank_volume_accumulator
   - waste_water_volume_accumulator
-  - recirculation_water_volume_accumulator
\ No newline at end of file
+  - recirculation_water_volume_accumulator
+
+WM_PULSE:
```
[the remark this answers is not in the document]
Comment (pranay2811) [reply ontology/yaml/resources/METERS/entity_types/WM.yaml:64]: Added guid for WM_PULSE

File: ontology/yaml/resources/METERS/entity_types/WM.yaml [hunk 2]
```diff
@@ -59,4 +59,15 @@ WM_WASTE_NONCANONICAL_1:
...
+  description: "Water meter that operates by counting pulses, interpreted into flow and accumulated volume."
+  implements:
+  - WM
+  is_canonical: true
+  uses:
+  - water_volume_accumulator
+  opt_uses:
+  - water_flowrate_counter
```
[the remark this answers is not in the document]
Comment (pranay2811) [reply ontology/yaml/resources/METERS/entity_types/WM.yaml:72]: Made water_flowrate_counter required field @shambergoldstein

File: ontology/yaml/resources/METERS/entity_types/WM.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (shambergoldstein) [reply ontology/yaml/resources/METERS/entity_types/WM.yaml:64]: please add a tab before the guid (put it in line with description, implements, etc.)

[the remark this answers is not in the document]
Comment (pranay2811) [reply ontology/yaml/resources/METERS/entity_types/WM.yaml:64]: done

**Review by shambergoldstein — APPROVED** [review]

**Pull request merged**